### PR TITLE
tests/nested/manual/core20-4k-sector-size: do not assume we have xxd

### PR DESCRIPTION
**Pull request by valentindavid**
xxd is from vim. od from coreutils.

Thanks for helping us make a better snapd!
Have you signed the [license agreement](https://www.ubuntu.com/legal/contributors) and read the [contribution guide](https://github.com/snapcore/snapd/blob/master/CONTRIBUTING.md)?

### CODE REVIEW

File: tests/nested/manual/core20-4k-sector-size/task.yaml [hunk 1]
```diff
@@ -50,7 +50,11 @@ execute: |
     remote.exec "test -e /sys/kernel/security/tpm0/binary_bios_measurements"
 
     echo "and secure boot is enabled on the nested vm"
-    remote.exec "xxd /sys/firmware/efi/efivars/SecureBoot-8be4df61-93ca-11d2-aa0d-00e098032b8c" | MATCH "00000000: 0600 0000 01\s+....."
+    if remote.exec "command -v xxd"; then
```
Comment (zyga) [inline tests/nested/manual/core20-4k-sector-size/task.yaml:53]: Nitpick, perhaps we can standardize on `od` instead? The output is GREP friendly and we'd have fewer conditional pieces.

Comment (alfonsosanchezbeato) [reply tests/nested/manual/core20-4k-sector-size/task.yaml:53]: +1 on using od that should be available everywhere

Comment (valentindavid) [reply tests/nested/manual/core20-4k-sector-size/task.yaml:53]: Good idea.

Comment (valentindavid) [reply tests/nested/manual/core20-4k-sector-size/task.yaml]: I have forced push a replacement since this is such a small commit.

**Review by zyga — APPROVED** [review]
Looks great

**Review by ernestl — APPROVED** [review]
Thanks

**Review by alfonsosanchezbeato — COMMENTED** [review]
LGTM

**Pull request merged**